zulipterminal/ui_tools/boxes.py

```diff
@@ -734,6 +736,19 @@ def soup2markup(self, soup: Any, **state: Any) -> List[Any]:
                 markup.extend(self.soup2markup(element, **state))
             elif element.name == 'table':
                 markup.extend(render_table(element))
+            elif element.name == 'time':
+                # Render time in current user's local time zone.
+                timestamp = element.get('datetime')
+                if not timestamp:
+                    continue
+                utc_time = parse(timestamp)
+                local_time = utc_time.astimezone(get_localzone())
+                # TODO: Address 12-hour format support with application-wide
+                # support for different formats.
```

Comment:
Do you mean, to allow users to customize their date/time output beyond the `strftime` format string below?

Was this particular string inspired by the webapp version? What do you think about adding the timezone (`%Z`) in parentheses at the end, to indicate it's in the user's local time?

Reply:
Yes. Ideally, we'd switch the formatting according to the application-wide 12 hour/24 hour time format setting, much like the web app.

Yes, the string is inspired by the web app version. Though, the timezone addition seems plausible. I have added it in the latest update.

Reply:
Want to add an issue to discuss/explore this?

Reply:
Filed #770.